packages/backend-common/src/sqs.ts

```diff
@@ -80,6 +80,19 @@ export const generateOutputSignedUrlAndSendMessage = async (
 		7,
 	);
 
+	const combinedOutputKey = `combined/${s3Key}.json`;
+
+	const combinedUrl = await getSignedUploadUrl(
```

Comment:
would be good if `getSignedUploadUrl` could be refactored to take a single parameter which is an object, so that the **many** inputs are forced to be labelled at the call site(s)

Reply:
Agreed, it's quite confusing already. I'll do it in a follow up PR